scripts/jenkins_ci/integration_test_common.groovy

```diff
@@ -0,0 +1,171 @@
+test_case_names = ["simple", "cdc", "multi_capture", "split_region", "row_format"]
+
+def prepare_binaries() {
+    stage('Prepare Binaries') {
+        def prepares = [:]
+
+        prepares["download third binaries"] = {
+            node ("${GO_TEST_SLAVE}") {
+                container("golang") {
+                    def ws = pwd()
+                    deleteDir()
+
+                    sh """
+                        mkdir -p third_bin
+                        mkdir -p tmp
+                        curl ${FILE_SERVER_URL}/download/builds/pingcap/tidb/700d9def026185fe836dd56b0c39e0b4df3c320b/centos7/tidb-server.tar.gz | tar xz -C ./tmp bin/tidb-server
```

Comment:
maybe we can refine the binary used here. (we can refine it in another PR)
- fetch `tidb_sha1` from `${FILE_SERVER_URL}/download/refs/pingcap/tidb/${TIDB_BRANCH}/sha1")`
- and use any commit hash by `branch` or `release tag`

Reply:
fine i will refine it in another PR